fastlane/Fastfile

```
@@ -0,0 +1,103 @@
+def latest_googleplay_version_code
+#   productionVersionCodes = google_play_track_version_codes(track: 'production')
+#   betaVersionCodes = google_play_track_version_codes(track: 'beta')
+#   alphaVersionCodes = google_play_track_version_codes(track: 'alpha')
+  internalVersionCodes = google_play_track_version_codes(track: 'internal')
+
+  # puts version codes from all tracks into the same array
+  versionCodes = [
+#     productionVersionCodes,
+#     betaVersionCodes,
+#     alphaVersionCodes,
+    internalVersionCodes
```

rubocop (1.69.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)